app/src/main/java/xyz/ivaniskandar/shouko/feature/GAKeyOverrider.kt

```diff
@@ -136,7 +136,7 @@ class GAKeyOverrider(
                     // Why even
                     return
                 }
-                if (service.isPackageInstalled(GOOGLE_PACKAGE_NAME)) {
+                if (service.isPackageInstalled(GOOGLE_PACKAGE_NAME) && service.isPackageInstalled(GOOGLE_ASSISTANT_PACKAGE_NAME)) {
```

Comment:
Is this actually needed?

Reply:
In my case I had uninstalled the Google Assistant app using `adb` and I found that the accessibility event for the window state changing wasn't being triggered.